ocean_provider/file_types/definitions.py

```diff
@@ -72,7 +72,10 @@ def check_details(self, with_checksum=False):
                 if result:
                     status_code = result.status_code
                     headers = copy.deepcopy(result.headers)
-                    files_url = copy.deepcopy(result.url)
+                    if "arweave" in result.url:
+                        files_url = ""
```

Comment:
can we get at least Content-size ?

Reply:
I return content-length and content-type, these come from headers.

Reply:
I assume you referred to Content-Length